### DISCUSSION TITLE
Does this work on a AWS EC2 instance?

### DISCUSSION THREAD

**Discussion by timbrowning327** [JuanBindez/pytubefix#313] [General]
My local machine runs the code no problem and can scrape the transcript but running it on the ec2 instance it returns null. Here is the code - 

def get_youtube_transcript(video_url):
    try:
        yt = YouTube(video_url, on_progress_callback=on_progress)
        transcript = yt.captions['a.en'].generate_srt_captions()
        cleaned_transcript = re.sub(r'\d+\s*\n|\d{2}:\d{2}:\d{2},\d{3}\s*-->\s*\d{2}:\d{2}:\d{2},\d{3}\s*\n', '',
                                    transcript)
        cleaned_transcript = " ".join([line.strip() for line in cleaned_transcript.splitlines() if line.strip()])
        return cleaned_transcript
    except Exception as e:
        print(f"Error fetching transcript for video {video_url}: {str(e)}")
        return None

Is it blocked on EC2?

**Comment by timbrowning327** [comment]
This works on GCP but not AWS. AWS must be blocking this package in some way

**Reply by kevin-weitgenant** [reply]
mine doesnt work on GCP. I didnt try in AWS. I deployed to clouder runner and used proxy. still not works  **Could not retrieve a transcript for the video https://www.youtube.com/watch?v=YJthAe9wBfw! This is most likely caused by:**
